lib/contacts.py

```diff
@@ -63,6 +64,14 @@ def import_file(self, path):
         self.update(d)
         self.save()
 
+    def export_file(self, fileName):
+        try:
+            with open(fileName, 'w+') as f:
+                json.dump(self, f, indent=4, sort_keys=True)
```

Comment:
btw you can add a "success" pop-up as you originally did, I think that's a good idea. A clean way to do it is with a `try-except-else`, in the `else` branch. If you do, add that to that importing too. :P